### DISCUSSION TITLE
Clarification on recommended upscalers

### DISCUSSION THREAD

**Discussion by system1system2** [Coyote-A/ultimate-upscale-for-automatic1111#16] [Q&A] [answered]
First of all, thanks for this amazing upscaling tool. 

Q: in your FAQ, you say:

> Our recommendation - ESRGAN to photorealistic images

but `ESRGAN` is not an option in A1111, not even in your own screenshots. Do you mean `ESRGAN_4x`?

**Comment by Jopezzia** [comment, marked as answer]
Yes, I mean ESRGAN_4x, just shortened the name
